test/euros/http_test.exs

```diff
@@ -0,0 +1,7 @@
+defmodule Euros.HTTPTest do
+  use ExUnit.Case, async: true
+
+  test "fetch_pages" do
+    assert %{body: _, headers: _, request_url: _, status_code: 200} = Euros.HTTP.fetch_pages("http://example.com")
```

Comment:
What will you do case of can not access at "http://example.com" ?
I think you should edit this code like using mock.

Reply:
I changed this code like access the server for test.
That server is `inets` from erlang. 
http://erlang.org/doc/apps/inets/http_server.html

And that server start at before begin test.